suites/pacific/rados/tier-3_rados_ssd-tests.yaml

```diff
@@ -162,14 +198,14 @@ tests:
               command: add
               service: osd
               pos_args:
-                - node4
-                - "/dev/sdc"
+                - node14
+                - "/dev/sdm"
```

Comment:
Do we have any code in place to ensure that /dev/sdm drive will not have an OSD added to it already and will be available for deployment every time the test runs?

Reply:
Added the check.